Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -84,6 +89,16 @@ @interface WMFExploreViewController ()
 
 @implementation WMFExploreViewController
 
+- (void)didReceiveMemoryWarning {
```

Comment:
This is just a testing stub. Will remove once we add a "Trending" section to the fee.